src/main/java/com/epam/library/dao/specification/BookSpecificationByPrice.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.library.dao.specification;
+
+import com.epam.library.entity.Book;
+
+public class BookSpecificationByPrice<T> extends AbstractBookSpecification<T> {
+
+    public BookSpecificationByPrice(T query) {
+        super(query);
+    }
+
+    @Override
+    public boolean specified(Book book) {
+        return book
```

Comment:
same, variable for price
because I want see variable types
what about if price is null?